MetaMask/metamask-extension
# feat: new asset page

## **Description**

Implements a new token details page that includes historical pricing and market data.

Introduces a chart.js dependency for graphs.

MMI is still on the old page in this PR, and will be migrated to the new page in a future PR.

## **Related issues**

Figma: https://www.figma.com/file/TfVzSMJA8KwpWX8TTWQ2iO/MMASSETS-15%3A-Asset-list-and-details?node-id=1719%3A48736&mode=dev

## **Manual testing steps**

- Click on a native or erc20 token on the home page
- Verify prices and balances display correctly
- Verify sending, bridging, swapping buttons work

## **Screenshots/Recordings**



### **Before**



### **After**


https://github.com/MetaMask/metamask-extension/assets/3500406/886a7496-49ff-40cb-ad27-e4565effd1f0




## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
